maxim/logger/llamaindex/client.py

```
@@ -0,0 +1,244 @@
+    """
+    global _INSTRUMENTED
+    if _INSTRUMENTED:
+        scribe().debug("[MaximSDK] LlamaIndex already instrumented")
+        return
+
+    def make_maxim_wrapper(original_method, base_op_name):
+        print(f"[MaximSDK] Creating wrapper for method: {base_op_name}")
+        print(f"[MaximSDK] Original method details: {original_method.__name__} from {original_method.__module__}")
+
+        @functools.wraps(original_method)
+        async def maxim_wrapper(self, *args, **kwargs):
```

Ruff (0.12.2)

`maxim_wrapper` is too complex (29 > 10)

(C901)